app/shared/reservation-form/catering/catering-form/CateringForm.spec.js

```diff
@@ -0,0 +1,142 @@
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+import React from 'react';
+import { Field } from 'redux-form';
+
+import CateringOrderTable from 'shared/catering-order-table';
+import ReduxFormField from 'shared/form-fields/ReduxFormField';
+import { UnconnectedCateringForm } from './CateringForm';
+import CateringMenu from './CateringMenu';
+
+describe('shared/reservation-form/catering/catering-form/CateringForm', () => {
+  const defaults = {
+    addOrRemoveItem: () => null,
+    cateringMenu: [{ id: 'c1', name: 'Drinks', products: [] }],
+    formValues: {
+      invoicingData: '12345',
+      message: 'Hola!',
+      order: {
+        2: 10,
+      },
+      servingTime: '15:45',
+    },
+    handleCancel: () => null,
+    handleSubmit: () => null,
+    initialized: true,
+    orderItems: [{ some: 'data' }],
+    priceListUrl: 'http://example.com/price-list-url',
+    updateOrder: () => null,
+  };
+  function getWrapper(props) {
+    return shallow(<UnconnectedCateringForm {...defaults} {...props} />);
+  }
+
+  it('renders a form.catering-form when form is initialized', () => {
+    const form = getWrapper().find('form.catering-form');
+    expect(form).to.have.length(1);
+    expect(form.prop('onSubmit')).to.equal(defaults.handleSubmit);
+  });
+
+  it('renders only an empty div when form not initialized', () => {
+    const wrapper = getWrapper({ initialized: false });
+    expect(wrapper.html()).to.equal('<div></div>');
+  });
+
+  describe('form fields', () => {
+    let fields;
+
+    beforeEach(() => {
+      fields = getWrapper().find(Field);
+    });
+
+    it('has servingTime field', () => {
+      const field = fields.filter({
+        component: ReduxFormField,
+        controlProps: {
+          step: 5 * 60,
+          value: defaults.formValues.servingTime,
+        },
+        label: 'Tarjoiluaika',
+        name: 'servingTime',
+        type: 'time',
+      });
+      expect(field).to.have.length(1);
+    });
+
```

Comment:
This will have conflicts with #251, I can fix them in that branch after this one is merged